user_templates/printer.cfg

```diff
@@ -190,16 +199,13 @@
 # ----------------------------------------------------------------------------------------
 
 
-# ------------------------------------------------------------------------------ ERCF ----> Select either all lines or none
+# --------------------------------------------------------------------------- MMU/ERCF ----> You can select multiple lines
 ### --------------------------------------------------------------------------------------
-### Klippain is designed to be used with the ERCF Happy Hare software: https://github.com/moggieuk/ERCF-Software-V3
-### Please run its own install script to get Happy Hare ready to be used
-### Then, include all of the following files that are created during the installation of Happy Hare to enable ERCF in Klippain
-# [include ercf_hardware.cfg]
-# [include ercf_parameters.cfg]
-# [include ercf_software.cfg]
-# [include ercf_menu.cfg]   # Optional: use it if you also have a Mini12864 display to add the ERCF menu entries
-# [include config/hardware/ercf.cfg]
+### Klippain is designed to be used with the MMU/ERCF HappyHare software backend: https://github.com/moggieuk/Happy-Hare
+### Please refer to the corresponding Klippain documentation: https://github.com/Frix-x/klippain/blob/main/docs/mmu.md
+# [include config/hardware/mmu.cfg]
+# [include mmu/optional/mmu_menu.cfg]         # Optional: use it if you also have a Mini12864 display and want to add the MMU/ERCF menu entries
+# [include mmu/optional/mmu_ercf_compat.cfg]  # Optional: use if you need the old Happy_Hare v1 retro compatibility (ERCF ref instead of MMU for some macro)
```

Comment:
Does this line is needed or can I remove it to simplify? I think it can be removed since Klippain has all the macro aligned with the newer MMU semantic and we don't need the compat

Reply:
mmu.cfg is needed :-)
ercf_compat..: It's for compatibility with the old macro from ette... it's not necessary for Klippain. Just for users who wants to use old ERCF typo. For exemple ERCF_HOME instead of MMU_HOME.
For the display I think it's needed for users with mini display. But I never test it.

Reply:
I was only talking about the compat file, not the other one :) Sorry it selected all the lines at once ^^

So I'll remove it since it's not directly needed for Klippain and will simplify. An user can add it to its overrides if he wants to use these names or adapt his own macros (that are also likely already in overrides).